sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -196,7 +205,12 @@ public void setSampling(Double sampling) {
   }
 
   public SentryOptions() {
-    eventProcessors.add(new MainEventProcessor(this));
+    SentryStackTraceFactory sentryStackTraceFactory = new SentryStackTraceFactory(this.inAppPrefix);
```

Comment:
Can we bury this into `MainEventProcessor` on a ctor overload? One takes everything, the second passes new instances like this?
`SentryStackTraceFactory` would take either the `List<String>`s for InAppInclude and InAppExclude or the whole `Option`.